src/components/organisms/KeyboardShortcuts/styled.tsx

```diff
@@ -0,0 +1,80 @@
+import {Modal} from 'antd';
+
+import styled from 'styled-components';
+
+import Colors from '@styles/Colors';
+
+export const StyledModal = styled(Modal)`
+  .ant-modal-close-icon {
```

Comment:
Is this Modal different than the others used? Maybe we could modify the global antd theme in order to keep the modals consistent, wdyt?

Reply:
yes, we can move it to a shared stylesheet